GooglePayment/build.gradle

```diff
@@ -106,7 +101,7 @@ afterEvaluate {
 
                 pom {
                     name = 'google-payment'
-                    packaging = 'jar'
+                    packaging = 'aar'
```

Comment:
Why did this change?

Reply:
It actually wouldn't deploy to local maven when set to `jar`. Since AAR is more accurate, we went with that and it worked. It may have worked previously with Maven Central though.